parser/schema/avro_schema

```diff
@@ -0,0 +1,39 @@
+{
+	    "type": "record",
+	    "name": "simple",
+	    "namespace": "org.hamba.avro",
+	    "values": "string",
+	    "fields": [
+	    	{
+			"name": "Metrics", "type": {
+				"type": "array",
+				"items": {
+					"type": "record",
+					"name": "metrics",
+					"fields": [
+                        {
+							"name": "Time",
+							"type": "long.timestamp-micros"
+							
+						},
+						{
+							"name": "Metadata",
+							"type": {
+								"type": "map",
+								"values": "string"
+							}
+						},
+                        {
+							"name": "Data",
+							"type": {
+								"type": "map",
+								"values": ["string", "int", "long", "float", "double" ]
+							}
+						}
+						
+					]
+				}
+			}
+		}
+	]
+}
```

Comment:
This is the same file as `encoder/schema/avro_schema` as far as I can tell, is there a reason for it being duplicated, or could we have only one file, but referenced for purposes from both encoder and parser? That way we don't have any "Oh I forgot to update the other schema file"-mistakes :)

Reply:
I  will follow the recommendation @KristianLyng  here.